tests/v2/test_waku_swap.nim

```diff
@@ -117,26 +121,31 @@ procSuite "Waku SWAP Accounting":
     await server.mountStore(store=StoreQueueRef.new())
     await client.mountSwap(swapConfig)
     await client.mountStore()
+    client.mountStoreClient()
 
-    client.wakuStore.setPeer(server.peerInfo.toRemotePeerInfo())
     client.wakuSwap.setPeer(server.peerInfo.toRemotePeerInfo())
```

Comment:
Beyond the scope of the PR but should we also add `server.setSwapPeer` instead of `server.wakuSwap.setPeer`? Can see we already have `setLightPushPeer` and `setPeerExchangePeer`.

Reply:
One protocol at a time. Check PR #1308 for lightpush protocol.

Reply:
great, just saw it!